charter.md

```diff
@@ -8,7 +8,7 @@ solution will be implementable in both browser and non-browser endpoints.
 The work focuses on building protocol mechanisms for publication of media
 and means to identify and receive the media.
 
-The media publication protocol will be a push protocol for sending
+The media publication protocol will be a protocol for sending
```

Comment:
"protocol will be a protocol" really stands out now.  It might be worth reframing this sentence.

Reply:
```suggestion
The media publication protocol will enable sending
```
Maybe...